extensions/wikia/AchievementsII/services/AchAwardingService.class.php

```diff
@@ -434,13 +435,13 @@ private function processInTrackPicture() {
 
 							wfProfileOut( __METHOD__ . '::apiCall' );
 
-							if ( is_array( $res['query']['imageusage'] ) ) {
-								$imageUsageCount = count( $res['query']['imageusage'] );
+							if( is_array( $res[ 'query' ][ 'imageusage' ] ) ) {
+								$imageUsageCount = count( $res[ 'query' ][ 'imageusage' ] );
 							}
-						} catch ( Exception $e ) {
+						} catch( Exception $e ) {
 						};
 
-						if ( $imageUsageCount < $imageUsageLimit ) {
+						if( $imageUsageCount < $imageUsageLimit ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS134)